genome_assembly/hic-scaffolding.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Genome scaffolding with Hi-C on Galaxy Australia
+type: genome-scaffolding
+contributors: [VGP Project, Luke Silver, Anna Syme]
```

Comment:
Should `VGP` be an affiliation instead?

Reply:
I think contributor is better, as they are really the authors of the base workflow.